samples/multicore/hello_world/README.rst

```diff
@@ -51,12 +86,16 @@ After programming the sample to your development kit, complete the following ste
 
             .. code-block:: console
 
-               *** Booting Zephyr OS build v2.7.99-ncs1-2193-gd359a86abf14  ***
+               *** Booting nRF Connect SDK zephyr-v3.5.0-3517-g9458a1aaf744 ***
```

Comment:
is this real output you got with zephyr-v3.5.0 in the name? That should not be there

Reply:
this is the real output